proof/invariant-abstract/ARM_HYP/ArchVSpace_AI.thy

```diff
@@ -2656,6 +2656,37 @@ lemma invs_current_vcpu_update': "\<lbrakk> cur_vcpu_at v s \<and> invs s \<rbra
             valid_global_objs_def valid_global_vspace_mappings_def
            split: option.split)
 
+lemma vgic_update_sym_refs_hyp[wp]:
+  "\<lbrace>\<lambda>s. sym_refs (state_hyp_refs_of s)\<rbrace> vgic_update vcpuptr f \<lbrace>\<lambda>rv s. sym_refs (state_hyp_refs_of s)\<rbrace>"
```

Comment:
These would be slightly nicer with invariant notation `f {| .. |}`, but I see they are just moved from ARchFinalise, so not really a must to change.